slapp/rois.py

```diff
@@ -97,13 +97,15 @@ def __init__(self,
                  image_shape: Tuple[int, int],
                  experiment_id: int,
                  roi_id: int,
-                 trace: Union[np.array, List[float]]):
+                 trace: Union[np.array, List[float]],
+                 is_binary: bool = False):
```

Comment:
This is fine, but also probably something that can be easily inferred as default behavior (instead of default as False)

Reply:
Leaving it, unless you really want something different